lua/tabnine/binary.lua

```diff
@@ -55,6 +55,9 @@ local function optional_args()
 	if config.log_file_path then
 		table.insert(args, "--log-file-path=" .. config.log_file_path)
 	end
+	if config.tabnine_enterprise_host then
+		table.insert(args, "--cloud2_url=" .. config.tabnine_enterprise_host)
```

Comment:
you need to pass  `--no_bootstrap` in this case

Reply:
No you don't. If you have cloud2url it won't bootstrap

Reply:
i'd suggest passing a different client name as well